ironfish-cli/src/commands/reset.ts

```diff
@@ -57,12 +43,12 @@ export default class Reset extends IronfishCommand {
 
     const message =
       '\nYou are about to destroy your node databases. The following directories and files will be deleted:\n' +
```

Comment:
This is a rather minor nitpick, but there's only one database being deleted now, so maybe a slight reword:
```suggestion
      '\nYou are about to destroy your local copy of the blockchain. The following directories and files will be deleted:\n' +
```